it/uk/gov/hmrc/mobileusercontact/test/MobileUserContactClient.scala

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmrc.mobileusercontact.test
+
+import org.scalatestplus.play.WsScalaTestClient
+import play.api.libs.ws.WSResponse
+
+import scala.concurrent.Future
+
+trait MobileUserContactClient extends WsScalaTestClient  {
```

Comment:
I think it's clearer for these things to be left inline (you don't have to click through from the test to see what the URL etc is).

Also a single trait with all the HTTP requests for the integration tests in it is likely to get very big in future.

Reply:
I agree that there is a trade off between making the code explicit (readable) vs maintainable. To my mind having less repeated code is better (as it is in production code) _as long as the intent is still clear_. Here I think the intention is clear when read in context. Also the implementation of the client isn't all that relevant to what the test is verifying. 

Perhaps you just wanted to see the client instantiated in the test?

Reply:
@howzat here is a comment that I wrote a few hours ago but forgot to send!

I suppose what I want to see in the test is what HTTP request is being made, because that's what the contract that's being tested is.

Also "all the HTTP requests that the tests make" isn't how I'd divide up responsibilities - I'd prefer to have some things that are responsible for different areas of functionality (e.g. "support requests" or "feedback submissions") and some things that are responsible for technical concerns (e.g. "HTTP client" or "HTTP header needed to trigger sandbox mode"). I think one thing that concerns me is these concerns being mixed.